sktree/datasets/hyppo.py

```diff
@@ -110,8 +133,9 @@ def make_trunk_classification(
     """
     rng = np.random.default_rng(seed=seed)
 
-    mu_1 = np.array([1 / np.sqrt(i) for i in range(1, n_dim + 1)])
-    mu_0 = m_factor * mu_1
+    mu_1 = m_factor * np.array([1 / np.sqrt(i) if i < 257 else 0 for i in range(1, n_dim + 1)])
```

Comment:
Where did the 257 come from? Is that the hyperparameter to limit the number of informative dimensions?

In #200 I implemented `n_dim` vs `n_informative`, which I think is more general and self-documenting.